File: help/eng_gb/exercicios_da_ficha_9/Ficha09_010/Ficha09_010.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Number : " );// print text
    num = parseInt(prompt("Number : " ,"0") );
    
    document.write(num + "\n");// print input 
    document.write("Digits = " );
    while(num > 0 ){
        document.write(num % 10 + " " );
        num /=10;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_010/Ficha09_010.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha09_010 {

    //Main Function
    static void Main() {
        Console.Write("Number : " );
        int num = int.Parse(Console.ReadLine());
        Console.Write("Digits = " );
        while(num > 0 ){
            Console.Write(num % 10 + " " );
            num /=10;
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:12
